apps/cli/package.json

```diff
@@ -9,19 +9,27 @@
     "webstudio": "./bin.js"
   },
   "scripts": {
-    "dev": "build-package --watch",
-    "build": "build-package"
+    "build:templates": "tsx ./templates/build.ts",
+    "dev": "pnpm build:templates && build-package --watch",
```

Comment:
Why do we need dev command here?

Reply:
I am actually using this, to test the `cli` while building itself by using `npm link`. Comes in handy while making changes and checking the same in real time.

Reply:
You could use tax for testing. No need in watcher.

Reply:
Cool, removing it then. Didn't know about this one